src/Symfony/Component/Translation/composer.json

```diff
@@ -21,6 +21,7 @@
         "symfony/translation-contracts": "^2.3|^3.0"
     },
     "require-dev": {
+        "nikic/php-parser": "^4.13",
```

Comment:
maybe also list this in suggest for discoverability? and mention that this is needed in the CHANGELOG?